code-style/php/README.md

```diff
@@ -0,0 +1,169 @@
+# PHP Coding Standards
+![](php-logo.png)
+
+Despite still being one of the most used languages to build web applications, PHP doesn't have the best reputation. There are a number of frameworks we use that are built with PHP. In order to avoid PHP's reputation of messy codebases, we should utilize [PHP_CodeSniffer](phpcs) to detect violations of a defined coding standards.
+
+### What Standard Should I use?
+
+Many frameworks have their own standards declared. We should use the framework's defined standard when possible. For instance, when developing a WordPress theme, you should utilize the [WordPress Standard](wp). If there is not a standard defined, or you are not using a framework, you should use the [PSR-2 Standard](psr-2).  Some standards are:
```

Comment:
Have you reached out to Naomi or Kaitlyn for a grammar review? I'm not sure if there is an "official" style guild for standard contributions, but Naomi sent me a doc with style recommendations. I'll send it to you.